src/containers/WelcomePage/WelcomePage.tsx

```diff
@@ -23,17 +22,15 @@ import FrontpageSubjects from './FrontpageSubjects';
 import { FILM_PAGE_PATH } from '../../constants';
 import SocialMediaMetadata from '../../components/SocialMediaMetadata';
 import config from '../../config';
-
-import { getLocaleUrls } from '../../util/localeHelpers';
-import { LocationShape } from '../../shapes';
 import BlogPosts from './BlogPosts';
 import WelcomePageSearch from './WelcomePageSearch';
 import { toSubject, toTopic } from '../../routeHelpers';
 import { getSubjectById } from '../../data/subjects';
+import { LocaleType } from '../../interfaces';
 
-const getUrlFromSubjectId = subjectId => {
+const getUrlFromSubjectId = (subjectId: string) => {
   const subject = getSubjectById(subjectId);
-  return toSubject(subject.id);
+  return toSubject(subject!.id);
```

Comment:
Liker ikke helt `!` siden det ut fra koden ser ut som at subject i teorien kan være undefined. Går det an å skrive om dette så man unngår `!`? Så fremt det ikke blir masse ekstra arbeid og rot så klart 😅

Reply:
Det er nok mulig, men ganske upraktisk. Akkurat nå sendes det kun inn hardkodede ID'er til funksjonene som bruker `!`, så vi er forsåvidt sikre på at de eksisteren da alle subjects også er hardkodet. Det er dog ingen enkel måte å ekstrahere ID'ene til disse subjectene ut i en type på. Og dersom vi hadde gjort det tror jeg autocomplete hadde tatt en liten hit på grunn av det. Vi kan sikkert skrive kode for å håndtere situasjoner der et subject ikke eksisteren, men med mindre noen sletter et hardkodet fag fra `subjects.ts` vil det ikke skje.

Reply:
Er det mulig å legge inn noen ekstra if-statements for å gjøre dette bulletproof? Hvis ikke så sier jeg LGTM 💯

Reply:
Jobb gjort. Skal fungere likt.